misago/users/utils.py

```diff
@@ -1,5 +1,30 @@
 import hashlib
 
+from django.utils import timezone
+
+from misago.users.models.deleteduser import DeletedUser
+
 
 def hash_email(email):
     return hashlib.md5(email.lower().encode('utf-8')).hexdigest()
+
+
+def record_user_deleted_by_self():
+    return DeletedUser.objects.create(
+        deleted_by=1,
+        deleted_on=timezone.now()
```

Comment:
It's unnecessary to do `deleted_on=timezone.now()` in utility functions, because model itself already sets those on creation.